docs/reference/troubleshooting/troubleshooting-unbalanced-cluster.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[troubleshooting-unbalanced-cluster]]
+== Troubleshooting an unbalanced cluster
+
+Since 8.6 Elasticsearch is balancing shards across data tier to achieve a good compromise between
+* shard count
+* disk usage
+* ingest load forecast for write indices
+
+Please note, elasticsearch does not take into account amount or complexity of search queries when rebalancing shards.
+
+There is no guarantee that individual components are going to be evenly spread across the nodes.
+This might appear as some of the nodes have less shards allocated or using less disk
+as long as they are assigned with the shards with greater ingest load forecast.
+
+You might get a detailed output of assigned workloads per node using <<cat-allocation,cat allocation command>>.
+
+[source,console,id=cat-allocation-example]
+--------------------------------------------------
+GET /_cat/allocation
+--------------------------------------------------
+
+The API returns the following response:
+
+[source,text]
+--------------------------------------------------
+shards shards.undesired write_load.forecast disk.indices.forecast disk.indices disk.used disk.avail disk.total disk.percent host      ip        node    node.role
+     1                0                 0.0                  260b         260b    47.3gb     43.4gb    100.7gb           46 127.0.0.1 127.0.0.1 CSUXak2 himrst
+--------------------------------------------------
```

Comment:
Could we turn this into a docs test, i.e. assert that the structure of the output of this API is as expected?